domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/filesystem/symbolicLinkResolving.kt

```diff
@@ -0,0 +1,101 @@
+package ch.tutteli.atrium.domain.robstoll.lib.creating.filesystem
+
+import ch.tutteli.atrium.assertions.Assertion
+import ch.tutteli.atrium.assertions.AssertionGroup
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.reporting.RawString
+import ch.tutteli.atrium.translations.DescriptionPathAssertion.FAILURE_DUE_TO_LINK_LOOP
+import ch.tutteli.atrium.translations.DescriptionPathAssertion.HINT_FOLLOWED_SYMBOLIC_LINK
+import java.io.IOException
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.*
+
+inline fun explainForResolvedLink(path: Path, block: (realPath: Path) -> Assertion): Assertion {
+    val hintList = LinkedList<Assertion>()
+    val realPath = addAllLevelResolvedSymlinkHints(
+        path.toAbsolutePath(),
```

Comment:
I would also move the `toAbsolutePath` call into addAllLevelResolvedSymlinkHints the one which currently expects two arguments

Reply:
Done.